### DISCUSSION TITLE
rdk_doc/Application_case/amr

### DISCUSSION THREAD

**Discussion by giscus** [D-Robotics/rdk_doc#58] [General]
# rdk_doc/Application_case/amr

自主移动机器人（Autonomous Mobile Robot，AMR）是一类能够在环境中自主导航和执行任务的机器人。AMR 不同于自动导航车辆 (AGV)，AGV依赖于轨道或预定义路线，并且通常需要操作员监督。AMR 通过多传感器融合、人工智能、机器学习等多种技术来理解环境并在其中导航，不受有线电源的限制。因其高度灵活的移动能力和智能导航系统，AMR在工业自动化、物流、医疗等领域得到了广泛应用。

https://developer.d-robotics.cc/rdk_doc/Application_case/amr

**Comment by 363546178** [comment]
单目相机的标定参数部分有误,修改后如下:
#设置棋盘格标定板参数，其中targetCols，targetRows分别为每列、每行的内点数，如格子数为9列x8列，则内角点数为8列x7列。
target_type: 'checkerboard' 
targetCols: 11 
targetRows: 8  
rowSpacingMeters: 0.02        
colSpacingMeters: 0.02

**Reply by wunuo1** [reply]
没理解兄弟这说的是哪里有问题？文中提供的只是说明，需要根据实际标定板修改参数

**Reply by 363546178** [reply]
抱歉忘了详细说哈，文中是checkboard，但kalibr中要求参数名是checkerboard哈😊参数的确是根据实际标定板来修改的，但target_type只能checkerboard和aprilgrid二选一🐞

**Reply by WuChao-2024** [reply]
感谢您的反馈，如果您有更多问题，欢迎通过社区论坛与我们联系。
